client/components/buttons/ReviewButton.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import './buttons.scss';
+
+const propTypes = {
+  recipe: PropTypes.objectOf(PropTypes.any).isRequired,
+  reviews: PropTypes.arrayOf(PropTypes.object).isRequired
+};
+
+class ReviewButton extends Component {
```

Comment:
This should be rewritten to be a stateless functional component seeing as it maintains no state of it's own nor does it use any of the lifecycle methods